### ISSUE TITLE
Fix: color contrast in Scrimba bootcamp description

### ISSUE THREAD

**Issue by michaella23** [ScrimbaBootcamp/progress-tracker#103]
On the homepage there is a section where we have a bootcamp description displayed in a grid format, but the text is black against a dark background and isn't readable. Change the text color and make sure the color contrast ratio meets [WCAG guidelines](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) (4.5 : 1 for regular text, 3 : 1 for large text). Here is a [color contrast checker browser extension](https://chromewebstore.google.com/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf) you can use if you don't have another one.

**Comment by Raehale** [comment]
Can I take this task?

**Comment by michaella23** [comment]
Yes, please do @Raehale !
